controllers/release/adapter_test.go

```diff
@@ -1254,6 +1265,19 @@ var _ = Describe("Release adapter", Ordered, func() {
 				Expect(conditionMsg).To(Equal("automated not set in status for automated release"))
 			})
 
+			It("returns invalid and an error if the error appears after 5 minutes of being created", func() {
+				adapter.release.SetCreationTimestamp(metav1.Time{Time: time.Now().Add(-7 * time.Minute)})
+				result := adapter.validateAuthor()
+				Expect(result.Valid).To(BeFalse())
+				Expect(result.Err).NotTo(HaveOccurred())
```

Comment:
Right here we check the error did not occur, but the test description says `and an error`